webapp/channels/src/components/permissions_gates/team_permission_gate/__snapshots__/team_permission_gate.test.tsx.snap

```diff
@@ -13,26 +13,27 @@ exports[`components/permissions_gates TeamPermissionGate should match snapshot w
     }
   }
 >
-  <Connect(TeamPermissionGate)
+  <Connect(Component)
     permissions={
       Array [
         "test_team_permission",
       ]
     }
     teamId="invalid_id"
   >
-    <TeamPermissionGate
+    <Memo(TeamPermissionGate)
       dispatch={[Function]}
       hasPermission={false}
-      invert={false}
       permissions={
         Array [
           "test_team_permission",
         ]
       }
       teamId="invalid_id"
-    />
-  </Connect(TeamPermissionGate)>
+    >
+      <p />
```

Comment:
This snapshot is now showing the content when it wasn't. It is due to my other comment, so make sure you re-take the snapshots after the fix.

Reply:
will do :+1: